src/app/components/MediaLoader/configs/portraitClipMedia.ts

```diff
@@ -6,93 +6,47 @@ import {
   ConfigBuilderReturnProps,
   PlaylistItem,
 } from '../types';
-import shouldDisplayAds from '../utils/shouldDisplayAds';
-import AUDIO_UI_CONFIG from './constants';
 
 const DEFAULT_WIDTH = 512;
 
+export const setImageWidth = (url?: string) =>
+  url?.replace('{width}', String(DEFAULT_WIDTH));
+
 export default ({
   blocks,
   basePlayerConfig,
-  adsEnabled = false,
-  showAdsBasedOnLocation = false,
 }: ConfigBuilderProps): ConfigBuilderReturnProps => {
-  const firstBlock = filterForBlockType(
-    blocks,
-    'portraitClipMedia',
-  ) as PortraitClipMediaBlock;
-
-  if (!firstBlock) {
-    return {
-      mediaType: 'video',
-      playerConfig: basePlayerConfig,
-      showAds: false,
-      orientation: 'portrait',
-    };
-  }
+  const { model }: PortraitClipMediaBlock =
+    filterForBlockType(blocks, 'portraitClipMedia') ?? {};
 
-  const portraitClipMediaBlocks: PortraitClipMediaBlock[] = filterForBlockType(
-    blocks,
-    'portraitClipMedia',
-    { returnAllMatchingBlocks: true },
-  ) as PortraitClipMediaBlock[];
+  const { video, images = [] } = model || {};
```

Comment:
Do we still need this if we've already returned `{}` on line 19?
```suggestion
  const { video, images = [] } = model;
```